shpc/main/container/docker.py

```diff
@@ -214,6 +215,26 @@ def install(
         # If there's a tag in the name, don't use it
         name = name.split(":", 1)[0]
 
+        # Option to create wrapper scripts for commands
+        module_dir = os.path.dirname(module_path)
```

Comment:
The section here and below look hugely redundant! Can we not have a shared function for the container base to add wrapper scripts?

Reply:
Ok, with the latest commit ( cf5b0c37783203463985a371a5b06c1d29befb16 ) I have created a new function , `_generate_wrapper_scripts`, in `shpc/main/container/base.py`, that gets called by the singularity and docker specific methods.

I am going to resolve this conversation and comment on the new bit of code